### ISSUE TITLE
Doesn't support array of strings passed to t function

### ISSUE THREAD

**Issue by tschanz-pcv** [i18next/i18next-parser#305]
## 🐛 Bug Report

In a `.tsx` file I have the following: `t(['flavor:welcome', 'welcome'])`
The array syntax can be found in the official docs, even with an example: https://www.i18next.com/overview/api#t

This syntax produces a warning `Key is not a string literal` and is ignored. The workaround with the comments is quite tedious and prone to be forgotten.

## To Reproduce

```tsx
import { FunctionComponent } from 'react';
import { useTranslation } from 'react-i18next';

const Demo: FunctionComponent = () => {
  const { t } = useTranslation();
  
  return (
    <span>{t(['flavor:welcome', 'welcome'])}</span>
  );
};
```

## Expected behavior

I would expect the array syntax to be supported.

## Your Environment

- *runtime version*: Node v14.16.1
- *i18next version*: 20.2.1 (i18next-parser: 3.11.0, react-i18next: 11.8.13)
- *os*: Mac

**Comment by karellm** [comment]
Indeed, feel free to submit a PR!

**Comment by karellm** [comment]
Fixed from `4.6.0`
